client/src/scripts/classes/Figure.js

```diff
@@ -96,13 +99,15 @@ export default class Figure extends THREE.Object3D {
         }
     }
 
-    capture() {}
+    capture() {
+    }
 
-    renew() {}
+    renew() {
+    }
 
-    canBuy() {}
+    static canBuy(data) {}
```

Comment:
is this ever use or will it be used?

Reply:
it is use in buy, I now that is too stupid because I use it only in buy but I think maybe I use it after